tech.feliciano.pocket-casts.yml

```
@@ -0,0 +1,63 @@
+app-id: tech.feliciano.pocket-casts
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
```

```suggestion
  - --share=network
  - --device=dri
```

will address

> libGL error: MESA-LOADER: failed to retrieve device information
libGL error: Version 4 or later of flush extension not found
libGL error: failed to load driver: i915
libGL error: failed to open /dev/dri/card0: No such file or directory
libGL error: failed to load driver: i965